crates/linter/src/plugin/compatibility/rules/php55/finally_feature.rs

```diff
@@ -0,0 +1,71 @@
+use indoc::indoc;
+
+use mago_ast::ast::*;
+use mago_php_version::PHPVersion;
+use mago_reporting::*;
+use mago_span::*;
+use mago_walker::Walker;
+
+use crate::context::LintContext;
+use crate::definition::RuleDefinition;
+use crate::definition::RuleUsageExample;
+use crate::rule::Rule;
+
+#[derive(Clone, Copy, Debug)]
+pub struct FinallyFeatureRule;
+
+impl Rule for FinallyFeatureRule {
+    fn get_definition(&self) -> RuleDefinition {
+        RuleDefinition::enabled("Finally Feature", Level::Error)
+            .with_maximum_supported_php_version(PHPVersion::PHP54)
```

Comment:
```suggestion
            .with_maximum_supported_php_version(PHPVersion::PHP55)
```

this has became exlusive now.